README.md

```diff
@@ -4,7 +4,16 @@
   <br /><br /><br />
 </div>
 
-Tachyon is a ZKP accelerator using CUDA.
+# Overview
+
+**Tachyon** is a Modular ZK Backend, powered by GPU.
+
+## Design Goals
+
+1. General Purpose: We want a code that enables switch different proving scheme.
+2. Easy to use: Switching code should be easy.
```

Comment:
2. Easy to Use: Achieving widespread adoption is essential for the success of any product. Consequently, one of the key focal points of the Tachyon project is to include offering packages for various programming languages and runtimes.